support/util/networking.go

```diff
@@ -70,6 +70,15 @@ func KASPodPortFromHostedCluster(hc *hyperv1.HostedCluster) int32 {
 	return 6443
 }
 
+// APIPortForLocalZone returns the port used by processes within a private hosted cluster
+// to communicate with the KAS via the api.<hc-name>.hypershift.local host.
+func APIPortForLocalZone(isRouteKAS bool) int32 {
+	if isRouteKAS {
```

Comment:
why does the LB needs to be 6443, why can't it also be 443?

If there's a real need for it to be 6443. Then lets do the other way around and consider 443 default and only use 6443 if isLBKAS(), so we make sure there's no possible impact/side effects.

Reply:
> why does the LB needs to be 6443, why can't it also be 443?

In case of LB KAS, traffic to the `.hypershift.local` endpoint is directed (through PrivateLink) to `service/kube-apiserver-private` which currently listens on port 6443:
https://github.com/openshift/hypershift/blob/a330a3f00979db1b6854b589fa6504e6186b48fe/control-plane-operator/controllers/hostedcontrolplane/kas/service.go#L176

> If there's a real need for it to be 6443. Then lets do the other way around and consider 443 default and only use 6443 if isLBKAS(), so we make sure there's no possible impact/side effects.

Yup, done.